release/models/system/openconfig-system.yang

```diff
@@ -362,6 +368,15 @@ module openconfig-system {
           value of the state/software-version leaf in the component
           of type OPERATING_SYSTEM.";
     }
+
+    leaf last-configuration-timestamp {
+      type oc-types:timeticks64;
+      units "nanoseconds";
+      description
+        "Indicates the timestamp at which the last configuration change
```

Comment:
@rolandphung what will happen if the configuration is not 'saved' or 'stored' to the device and the device is rebooted?  Perhaps we should clarify the description to specify the behavior or allow for differences.

Reply:
The current implementation in mind would have been that the data would have been lost, upon reboot the timestamp would update until the box reaches a steady state. Some metadata could be persisted to the filesystem to help identify if a config change was a real change or a reread of current config already accounted for. I suppose it depends on how strict we want this leaf to be, if its for people who just want to see if an unexpected configuration occurs by subscribing to the leaf then this is probably an ok edge case for them.

Reply:
For our operational use case, it is good to see the timestamp update if a reboot triggered a configuration change because the running configuration was not saved at the reboot time.  Can you confirm this is what you're expecting the behavior to be?

Reply:
Please update the description to require that the timestamp must only be incremented.    

This still leaves it up to the implementation on exactly how to behave in this scenario where 'unsaved' configuration is lost due to a reboot.  The implementation can leave the timestamp as the last time the config was changed, or update to the reboot time.

Reply:
Done